v19.1/managed-dockerize-your-app.md

```diff
@@ -0,0 +1,63 @@
+---
+title: Dockerize your Managed CockroachDB app
+summary: Learn how to build an application with Managed CockroachDB and orchestrate it using Kubernetes.
+toc: true
+build_for: [managed]
+---
+
+## Before your begin
+
+1. Follow Docker's [documentation](https://docs.docker.com/v17.12/docker-for-mac/install/) to install Docker.
+2. Build the sample Python application:
+
+    This tutorial uses the [sample To-Do app](https://github.com/cockroachdb/examples-python/tree/master/flask-sqlalchemy) application with Managed CockroachDB using [SQLAlchemy](https://docs.sqlalchemy.org/en/latest/). Refer to the [Build a Python app with Managed CockroachDB](managed-build-a-python-app.html) tutorial to build the sample application.
+
+## Create a Docker image for the application
+
+1. In the `flask-python` folder you created while [building the python application](managed-build-a-python-app.html), create the file names `dockerfile` and copy the following code into the file:
```

Comment:
"the file names `dockerfile`" -> "a file named `Dockerfile`".

We should just add this to the examples repo instead of asking the user to create it here.